lib/ses-to-cloudwatch/plugin.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace HM\AWS_SES_WP_Mail\CloudWatch;
+
+use Exception;
+use Aws\CloudWatchLogs\CloudWatchLogsClient;
+
+add_action( 'aws_ses_wp_mail_ses_sent_message', __NAMESPACE__ . '\\on_sent_message', 10, 2 );
+
+/**
+ * Called when the AWS SES plugin has sent an email.
+ *
+ * @param  AWS\Result $result
+ * @param  array $message
+ */
+function on_sent_message( $result, $message ) {
+	try {
+		// Lock the batch ID
+		$streams = cloudwatch_logs_client()->describeLogStreams([
+			'logGroupName'        => HM_ENV . '/ses',
+			'logStreamNamePrefix' => 'Sent',
+		])['logStreams'];
+
+		if ( ! $streams ) {
+			cloudwatch_logs_client()->createLogStream([
+				'logGroupName'  => HM_ENV . '/ses',
+				'logStreamName' => 'Sent',
+			]);
+		}
+		$params = [
+			'logEvents' => [
+				[
+					'timestamp' => time() * 1000,
+					'message'   => json_encode( $message ),
+				],
+			],
+			'logGroupName'  => HM_ENV . '/ses',
+			'logStreamName' => 'Sent',
+		];
+		if ( $streams ) {
+			$params['sequenceToken'] = $streams[0]['uploadSequenceToken'];
+		}
+		cloudwatch_logs_client()->putLogEventsAsync( $params );
+	} catch ( Exception $e ) {
+		trigger_error( 'Failed to send email to CloudWatch: ' . $e->getMessage(), E_USER_WARNING );
+	}
+}
+
+function cloudwatch_logs_client() {
```

Comment:
Can we call this `get_client` or something? This name is a bit weird. Also, needs phpDoc.